_i18n/de/resources/moneropedia/pruning.md

```diff
@@ -1,17 +1,17 @@
+terms: ["pruning", Pruning"]
+summary: "Funktion, die es Node-Betreibern erlaubt, lediglich 1/3 der Blockchain herunterzuladen und zu synchronisieren"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-'Pruning' allows node operators to save 2/3 of storage space while keeping the full transaction history. Pruning works by removing 7/8 of unnecessary ring signature data. The 1/8 remaining data will be available to the other nodes and will be used to sync with the network. Other pruned nodes will have a random 1/8 of the data, which they will also make available to the network. There are no privacy or security downsides when using a pruned node.
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+"Pruning" (eng. für Stutzen) ermöglicht es Node-Betreibern, 2/3 des Speicherplatzes einzusparen, während der gesamte Transaktionsverlauf bestehen bleibt. Pruning funktioniert durch das Entfernen von 7/8 der unwichtigen Ringsignatur-Daten. Das verbleibende Achtel bleibt für andere Nodes verfügbar und wird zum Synchronisieren mit dem Netzwerk genutzt. Andere Pruned-Nodes verfügen über ein zufälliges Achtel der Daten und stellen dies dem Netzwerk zur Verfügung. Es gibt beim Gebrauch eines Pruned-Nodes keine Nachteile bezüglich der Privatsphäre oder Sicherheit.
 
-There are three ways to prune:
+Es gibt drei Möglichkeiten des Prunings:
 
-- *Synchronize a pruned node from scratch:* You will download and sync only 1/3 of the full blockchain.
-- *Prune an existing node:* The already existing blockchain will be pruned, but this will not result in a smaller blockchain. Instead, it will mark parts of the file as free, so that future data will use that free space, causing the file to not grow until free space grows scarce.
-- *Create a new pruned blockchain from a full one:* A full blockchain that you already have will be used to create a new, pruned blockchain.
+- *Synchronisieren eines Pruned-Nodes von Grund auf:* Du synchronisierst und lädst lediglich 1/3 der gesamten Blockchain herunter.
+- *Pruning eines bestehenden Nodes:* Die bereits bestehende Blockchain wird gestutzt. Dies resultiert nicht in einer kleineren Blockchain, sondern wird Teile der Datei als leer markieren. Zukünftige Daten werden diesen nun freien Platz nutzen, damit die Datei nicht wächst, bis der Speicherplatz knapp wird.
+- *Erstellung einer neuen, gestutzten Blockchain aus einer kompletten Blockchain:* Eine komplette Blockchain, die du schon hast, wird zur Erstellung einer neuen, gestutzten Blockchain verwendet.
 
-Pruned nodes are very useful and preferable to @remote-nodes, but if possible, users should run a full node and opt for pruning only in case if necessary.
+Pruned-Nodes sind sehr nützlich und @Remote-Nodes vorzuziehen. Wenn es allerdings möglich ist, sollten Nutzer einen Full-Node betreiben und das Pruning nur im Falle der Notwendigkeit in Betracht ziehen.
```

Comment:
```suggestion
Pruned-Nodes sind sehr nützlich und @Remote-Nodes vorzuziehen. Wenn es allerdings möglich ist, sollten Nutzer einen Full-Node betreiben und das Pruning nur im Falle der Notwendigkeit in Betracht ziehen.
```

Translation is fine, but I think the English version should explain why pruned nodes are "very useful" (=> improves own security) and why users should still opt to run a full node instead (=> helps sharing of the blockchain with other nodes).

Reply:
Thanks for your suggestion. That's unrelated to this PR though, we can improve the original text later on.